package.json

```diff
@@ -56,9 +56,9 @@
     "copy-webpack-plugin": "^4.4.2",
     "css-loader": "^0.28.10",
     "duplexer": "^0.1.1",
-    "eslint": "4.5.0",
+    "eslint": "4.19.1",
     "eslint-loader": "^2.0.0",
-    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-markdown": "git://github.com/byzyk/eslint-plugin-markdown.git#5927ccb78aff8a5464663730800503c6de3704e1",
```

Comment:
Why aren't we using semver here?

Reply:
@dhruvdutt please see the third note in the description